readdirp.js

```diff
@@ -4,8 +4,10 @@ var fs        =  require('graceful-fs')
   , path      =  require('path')
   , minimatch =  require('minimatch')
   , toString  =  Object.prototype.toString
+  , si = typeof setImmediate !== 'undefined' ? setImmediate : function (fn) { setTimeout(fn, 0) }
```

Comment:
That's what I would do, and I do suggest you go ahead and do it in the PR. 

But we will have to wait to find out what @thlorenz thinks about it.